device_doctor/lib/src/android_device.dart

```diff
@@ -0,0 +1,195 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:meta/meta.dart';
+import 'package:retry/retry.dart';
+
+import 'device.dart';
+import 'process_helper.dart';
+import 'health.dart';
+import 'utils.dart';
+
+/// Constant battery health values returned from Android Battery Manager.
+///
+/// https://developer.android.com/reference/android/os/BatteryManager.html
+class AndroidBatteryHealth {
```

Comment:
Does iOS have a similar API? If so, we should see if we can generalize this.

Reply:
This is legacy code from agent. We may not need it since all phones are connected to hosts with full battery all the time. Just keep it here for potential future use. For ios case, I would leave the implementation when needed.

Reply:
If it's not used, we should remove it :-) Less code is easier to maintain, and worst case we can look up old git refs to view the agent code.

Reply:
SGTM to remove. Done.